packages/flutter_tools/lib/src/test/coverage_collector.dart

```diff
@@ -83,25 +83,29 @@ class CoverageCollector extends TestWatcher {
   Future<String> finalizeCoverage({
     coverage.Formatter formatter,
     Duration timeout,
+    String coverageDirectory,
   }) async {
     printTrace('formating coverage data');
     if (_globalHitmap == null)
       return null;
     if (formatter == null) {
       final coverage.Resolver resolver = coverage.Resolver(packagesPath: PackageMap.globalPackagesPath);
       final String packagePath = fs.currentDirectory.path;
-      final List<String> reportOn = <String>[fs.path.join(packagePath, 'lib')];
+      final List<String> reportOn = coverageDirectory == null
+        ? <String>[fs.path.join(packagePath, 'lib')]
+        : <String>[coverageDirectory];
```

Comment:
... and then this is `coverageDirectory.path`

Reply:
Done